TaskListRails/app/views/tasks/index.html.erb

```diff
@@ -1,9 +1,29 @@
 <%= image_tag "homer2.png", class: "homer" %>
+<h2>Dumb Stuff I Gotta Do</h2>
 <ul>
-    <% @tasks.each do |task| %>
-      <li><%= task.name %> <%= link_to "Show", "/tasks/#{task.id}" %>
-          <%= link_to "Delete", "/tasks/#{task.id}", :method => :delete,
-          :onclick => "return confirm('Are you sure?')"%></li>
+  <% @tasks.each do |task| %>
+    <% if task.completed_at == nil %>
+    <li>
+      <%= link_to "Mark Complete", "/tasks/#{task.id}/mark_complete", :method => :patch %>
+      <%= task.name %> <%= link_to "Show", "/tasks/#{task.id}" %>
```

Comment:
This code, except for the mark complete link, is pretty much exactly the same as the code below for the "Stuff I Did" section. Is there anything we could try doing to DRY out this code and remove the repetition?